op-chain-ops/genesis/config.go

```diff
@@ -732,6 +732,8 @@ type L1DependenciesConfig struct {
 
 	// DAChallengeProxy represents the L1 address of the DataAvailabilityChallenge contract.
 	DAChallengeProxy common.Address `json:"daChallengeProxy"`
+
+	ProtocolVersionsProxy common.Address `json:"protocolVersionsProxy"`
```

Comment:
Should there be any checks for this contract address in the `L1DependenciesConfig.CheckAddresses` method?

Reply:
I left that out for backwards compatibility. It's not strictly required and it would break a lot of existing deploy configs.